authors/apps/authentication/renderers.py

```diff
@@ -17,9 +18,9 @@ def render(self, data, media_type=None, renderer_context=None):
             # As mentioned about, we will let the default JSONRenderer handle
             # rendering errors.
             return super(UserJSONRenderer, self).render(data)
-
-
         # Finally, we can render our data under the "user" namespace.
+        token = generate_token(data)
```

Comment:
Sam here. Is this in the scope of this PR?

Reply:
I have fixed it, it was the one making my build fail